extension/ts/workers/finder/impl/util/getLinks.ts

```diff
@@ -0,0 +1,20 @@
+import {get$} from "../../../../services/request";
+import {Element} from "cheerio";
+
+interface GetLinksEnv {
+	baseUrl: string;
+	searchUrl: string;
+	aSelector: string;
+	maxResults: number;
+}
+
+const getLinks = async ({searchUrl, aSelector, baseUrl, maxResults}: GetLinksEnv): Promise<string[]> => {
+	const $ = await get$(searchUrl);
+	const paths: string[] = [];
+	$(aSelector).each((_, element: Element) => {
```

Comment:
just for my own understanding, does this go through every element it finds that matches the selector? we only filter/ shorten this in the next line, right?

Reply:
yeah this is just like jquery. it goes through all of them. There's probably a way to terminate early but I don't know how to use jquery and the results will only ever be a single page long